### PR TITLE
Refactor tests to use jest + @chainlink/test-helpers, add OVM support

### PR DESCRIPTION
**Pull request by krebernisak**
To get this running you will need to `yarn link` both `@chainlink/belt` and `@chainlink/test-helpers` from `feature/ovm-v10/contracts`.

### CODE REVIEW

File: test/behavior/LinkToken.behavior.ts [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  linkTokenFactory: ContractFactory,
+  linkReceiverFactory: ContractFactory,
+  Token677ReceiverMockFactory: ContractFactory,
+  notERC677CompatibleFactory: ContractFactory,
+  extraPublicABI: string[],
+) => {
+  describe('ERC677Token', () => {
+    const provider = setup.provider()
+
+    // @ts-ignore
```
Comment (HenryNguyen5) [inline test/behavior/LinkToken.behavior.ts:16]: We can use [this](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) now :D

**Review by HenryNguyen5 — DISMISSED** [review]
Looking really good, just one nitpick. Nice job! One other thing is file naming, do we really need the `.behaviour` extension when it already resides in its own directory? Seems a little redundant. I tend to do that type of naming when you have ambiguity from multiple types of files residing in the same location.

**Review by se3000 — COMMENTED** [review]
Love the move to Jest. Also, big fan of the shared behavior tests, we should use those more often. It does seem like CI is still hung up on some of the changes though. Gotta get that sorted before moving forward.

**Comment by se3000** [comment]
@krebernisak can you get that `yarn link` working on CI and document it in the README?

**Comment by krebernisak** [comment]
Closing this PR in favor of:

- [Refactor tests to use jest + @chainlink/test-helpers, v0.7 contracts](https://github.com/smartcontractkit/LinkToken/pull/24)
- [Optimism L2 integration](https://github.com/smartcontractkit/LinkToken/pull/23)

**Pull request closed without merging**